src/components/Button.tsx

```diff
@@ -9,8 +9,12 @@ interface ButtonProps {
 }
 
 const Button: React.FC<ButtonProps> = (ButtonProps) => {
-  const getButtonClasses = useMemo(() => {
-    let classes = "btn-default";
+  const getButtonClasses = useMemo(
+
+
+
+    () => {
+      let classes = "btn-default";
```

Comment:
I made changes to this button file to test out the pre commit hook.

You can use this file to test out your pre commit hook changes.
When successful, the pre commit hook will format it even if the user does not have es lint or prettier installed locally on their code editor. 

I also used [Option 1 ](https://prettier.io/docs/en/precommit.html) for my forked copy and recorded the pre commit results in the terminal. 

https://user-images.githubusercontent.com/67210629/177216862-41da756d-3df8-4268-a350-f1b79c75d84f.mov

Hope that helps!